docs/documentation/siddhi-5.x/user-guide-5.x.md

```diff
@@ -386,6 +386,25 @@ curl -X POST \
 	"power": 341
 }'
 ```
+You can use `sp, sps, SiddhiProcess, or SiddhiProcesses` names to view the deployment details of the `SiddhiProcess` custom objects.
+
+```
```

Comment:
Shall we differentiate the usages of the _kubectl gets_ under two types. Listing Siddhi Processes and getting the siddhi processes configs.

Reply:
What did you mean as getting the siddhi processes configs? Is that something like 

`kubectl describe sps example-siddhi-deployment`

Reply:
like `kubectl get sp example-siddhi-deployment -o yaml`